docs/Encyclopedia/pages/VCF_LiftOver.md

```diff
@@ -0,0 +1,27 @@
+# VCF LiftOver
+
+## Description ##
+
+VCF LiftOver is a pipeline used in GDC whole genome sequencing (WGS) harmonization.
+
+## Overview ##
+
+VCF LiftOver is a pipeline used for WGS harmonization of genomic profiling reports. Harmonization is performed with VCF LiftOver using tumor and normal alignments and generates combined nucleotide variation data.
+
+### Input
+
+* Submitted sequencing reads
+
+### Output
+
+* VCF (Data Type: Raw CGI Variant)
+
+## References ##
+
+1. []()
```

Comment:
```suggestion
```

Reply:
not sure if removing it completely works with suggestions.

Reply:
deleted the References section